tests/integration/cairo1/hint_locals/deploy/deploy_test.cairo

```diff
@@ -70,19 +54,13 @@ fn test_deploy_with_storage() {
     let class_hash = declare('with_storage').unwrap();
     assert(class_hash != 0, 'declared class_hash != 0');
 
-    let prepare_result = prepare(class_hash, ArrayTrait::new()).unwrap();
+    let prepare_result = prepare(class_hash, @ArrayTrait::new()).unwrap();
 
     assert(prepare_result.contract_address != 0, 'prepared contract_address != 0');
     assert(prepare_result.class_hash != 0, 'prepared class_hash != 0');
-    // TODO (1717): check the length of the array, this produces: error: Variable was previously moved
-    // assert(prepare_result.constructor_calldata.len() == 2_u32, 'constructor_calldata size == 2');
-
-    let prepared_contract = PreparedContract {
-        contract_address: prepare_result.contract_address,
-        class_hash: prepare_result.class_hash,
-        constructor_calldata: prepare_result.constructor_calldata
-    };
-    let deployed_contract_address = deploy(prepared_contract).unwrap();
+    assert(prepare_result.constructor_calldata.len() == 0_u32, 'constructor_calldata size == 0');
```

Comment:
```suggestion
    assert(prepare_result.constructor_calldata.len() == 2_u32, 'constructor_calldata size == 2');
    assert(*prepare_result.constructor_calldata.at(0_usize) == 1, 'constructor_calldata contents');
    assert(*prepare_result.constructor_calldata.at(1_usize) == 2, 'constructor_calldata contents');
```
nit, we could check this

Reply:
I'm sorry I added this comment in a wrong place, it should've been added above 🤦